10-indexing-records/README.md

```diff
@@ -71,13 +71,23 @@ Create a new file `indexer.py` and copy the following code:
 
 from __future__ import absolute_import, print_function
 
-def indexer_receiver(sender, json=None, record=None, index=None, doc_type=None):
+
+def indexer_receiver(
+    sender,
+    json=None,
+    record=None,
+    index=None,
+    doc_type=None,
+    arguments=None
```

Comment:
As pointed out by @kprzerwa the name is not fully clear. It could cause confusion with `*args`, but this is inherited from the signal signature so there's nothing to be done.